lib/webdriver.py

```diff
@@ -122,34 +127,52 @@ def get_reservations(self, account_monitor: AccountMonitor) -> List[JSON]:
         reservations = self._fetch_reservations(driver)
 
         driver.quit()
+        if self.display is not None:
+            self.display.stop()
+            logger.debug("Stopped virtual display successfully")
         return reservations
 
     def _get_driver(self) -> Driver:
+        is_docker = os.environ.get("AUTO_SOUTHWEST_CHECK_IN_DOCKER") == "1"
+
+        if is_docker:
+            try:
```

Comment:
Could we move this try-except to a separate function (e.g. `_start_display`). Then we could have the driver options set in this if-else statement and can even remove the separate `is_docker` variable.

Side note: If we have a `_start_display`, we might also want to include a `_stop_display` where the `_quit_driver(driver)` function will call `driver.quit()` and `self._stop_display` (to make the code more logically broken down).